src/Core/System/Snippet/SnippetFileHandler.php

```diff
@@ -29,6 +30,29 @@ public function writeJsonFile(string $path, array $content): void
         file_put_contents($path, $json);
     }
 
+    /**
+     * @return array<int, string>
+     */
+    public function findBundleSnippetFiles(BundleInterface $bundle): array
+    {
+        $storefrontSnippets = [];
+        if (is_dir($bundle->getPath() . '/Resources/snippet/')) {
+            $storefrontSnippets = $this->findSnippetFilesByPath($bundle->getPath() . '/Resources/snippet/');
+        }
+
+        $administrationSnippets = [];
+        if (is_dir($bundle->getPath() . '/Resources/app/*/src/')) {
+            $administrationSnippets = $this->findSnippetFilesByPath($bundle->getPath() . '/Resources/app/*/src/');
+        }
+
+        return array_merge($storefrontSnippets, $administrationSnippets);
+    }
+
+    /**
+     * @deprecated
```

Comment:
please write the proper deprecation tag. you can find other examples in the code.
something like:

```suggestion
     * @deprecated tag:v6.8.0 - Will be removed. Use `SnippetFileHandler::findBundleSnippetFiles(['administration'])` instead
```

or how ever the replacement would work now :slightly_smiling_face: 

same for the `findStorefrontSnippetFiles`

Additionally you need to trigger a deprecation notice within the method. 

```
        Feature::triggerDeprecationOrThrow(
            'v6.8.0.0',
            Feature::deprecatedMethodMessage(__CLASS__, __METHOD__, 'v6.8.0.0', 'SnippetFileHandler::findBundleSnippetFiles(['administration'])'),
        );
```

Disclaimer: code not tested :wink: